UI/ChatUI/ChatUI/ChatUI/Entities/Message.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+
+namespace ChatUI.Entities;
+
+public class Message
+{
+	public Message(string text, string contactName = default, DateTimeOffset timestamp = default, bool isMyMessage = true)
+	{
+		Text = text;
+		ContactName = contactName == default ? "Me" : contactName;
+		Timestamp = timestamp == default ? DateTimeOffset.Now : timestamp;
+		IsMyMessage = isMyMessage;
+	}
+
+	public string Text { get; init; }
+
+	public string ContactName { get; init; }
+
+	public DateTimeOffset Timestamp { get; init; }
+
+	public bool IsMyMessage { get; init; }
+
+	public bool IsLastInSequence { get; set; }
```

Comment:
If I'm right this boolean is no longer used (found it only in the template, but never set to `true`). As this tends to let users to think that this object is mutable (especially since it still has a `set`), I would remove it